docs/src/main/asciidoc/overview.adoc

```diff
@@ -637,7 +637,7 @@ Notice that the count property in the `x-death` header is a `Long`.
 == Error Channels
 
 Starting with version 1.3, the binder unconditionally sends exceptions to an error channel for each consumer destination and can also be configured to send async producer send failures to an error channel.
-See "`<<binder-error-channels>>`" for more information.
+See "`<<spring-cloud-stream-overview-error-handling>>`" for more information.
 
```

Comment:
Interesting, I was looking at how it's done in Kafka documentation and while it works in the [aggregated version](https://docs.spring.io/spring-cloud-stream/docs/current/reference/htmlsingle/#kafka-error-channels) it doesn't render properly in [Kafka-only reference](http://cloud.spring.io/spring-cloud-static/spring-cloud-stream-binder-kafka/2.1.2.RELEASE/single/spring-cloud-stream-binder-kafka.html#kafka-error-channels) - which makes sense.

This means that the change I made would also fix only the aggregated version - which is still better than the current state. 

Since there is a work going on already to get it fixed perhaps it doesn't make much sense to merge this one.